src/server/connection.cpp

```diff
@@ -186,6 +186,7 @@ void Connection::handle_timeout(boost::system::error_code ec)
     if (ec != boost::asio::error::operation_aborted)
     {
         boost::system::error_code ignore_error;
+        // NOLINTNEXTLINE(bugprone-unused-return-value)
         TCP_socket.cancel(ignore_error);
```

Comment:
As I understand Boost deprecated return value in both these cases (it used to return error in the past), so it is safe to ignore it.